app/javascript/components/individual/ApplicationStatusButtons.jsx

```diff
@@ -12,6 +12,7 @@ class ApplicationStatusButtons extends React.Component {
   }
 
   handleStatus(new_status) {
+    confirm("Are you sure you want to change this tenant's status?");
```

Comment:
if else case for this confirm statement. clicking cancel still causes the request to proceed

Reply:
fixed and used confirmmodal/deletemodal instead